src/server/req_runjob.c

```diff
@@ -1976,27 +1973,31 @@ req_defschedreply(struct batch_request *preq)
 void
 convert_job_to_resv(job *pjob)
 {
-	char *buf;
 	svrattrl *psatl;
 	unsigned int len;
 	pbs_list_head *plhed;
 	struct work_task *pwt;
 	struct batch_request *newreq;
+	char owner[PBS_MAXUSER + 1];
 
 	newreq = alloc_br(PBS_BATCH_SubmitResv);
 	if (newreq == NULL) {
-		(void)sprintf(log_buffer, "batch request allocation failed");
 		log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_JOB, LOG_ERR,
-			pjob->ji_qs.ji_jobid, log_buffer);
+			pjob->ji_qs.ji_jobid, "batch request allocation failed");
+		log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_JOB, LOG_ERR,
+			pjob->ji_qs.ji_jobid, "Could not create reservation from the job");
```

Comment:
Combine these log messages into one.

Reply:
i wanted to keep 2 separate log messages, but let us combine them into one.